Project6.md

```diff
@@ -16,11 +16,9 @@ Chose from the following list of topics:
       You can't use Django's caching interfaces to configure whole page caching with Redis without using additonal packages. Instead just directly call the Redis python client to store pages and later look them up. Think about how and when
       to invalidate the cache'd content (after a certain amount of time? when the DB changes? something else?).
     
-3. Unit tests
+3. Continuous Integration
     
-      Write unit tests for your models or experience services.Django has extensive support for writing unit tests
-      including running tests against a new 'test' database so that your main database isn't changed
-      by the test runs.
+      Use a continuous integration tool such as Travis CI to automate builds of your marketplace project. 
```

Comment:
Let's request that they use a CI tool to not only produce builds but also require that the CI runs unit tests along with any integration tests the team has written.